preproc/LED_charge_summary.ipynb

```diff
@@ -0,0 +1,1043 @@
+      "text/plain": [
+       "<Figure size 720x432 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(figsize=(10, 6))\n",
+    "ax      = plt.imshow(charge_mean)\n",
+    "\n",
+    "cbar    = plt.colorbar()\n",
+    "cbar.set_label('Mean charge (efine corrected)', fontsize = 14)\n",
+    "\n",
+    "plt.xlabel('Position sensor x', fontsize = 14)\n",
+    "plt.xticks(np.arange(0, 8, 1))\n",
+    "plt.ylabel('Position sensor y', fontsize = 14)\n",
+    "plt.yticks(np.arange(0, 8, 1))\n",
+    "\n",
+    "plt.gca().xaxis.set_label_position('top')\n",
+    "plt.gca().xaxis.tick_top()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "118835f9",
+   "metadata": {},
+   "source": [
+    "**TOFPET mean: sum of the corrected efine means for each channel**"
```

Comment:
Why is this number relevant? Are you summing the means of each channels? What does this number tell you?

Reply:
I have talked with Vicente, as we did this together and it is not relevant, so I have just remove it.